### PR TITLE
move deps forward

### PR DESCRIPTION
**Pull request by ahjohannessen**

### CODE REVIEW

**Comment by ahjohannessen** [comment]
@t3hnar WDYT?

File: .gitignore [hunk 1]
```diff
@@ -29,6 +29,12 @@ TAGS
 # Scala
 *.sc
 
+# Metals
+.metals/*
```
Comment (t3hnar) [inline .gitignore]: nice :)

**Comment by t3hnar** [comment]
Thanks @ahjohannessen

**Comment by ahjohannessen** [comment]
Thanks :) I'll update the Akka.Persistence PR with 7.1.0 as soon you publish.

**Pull request merged**